apps/frontend/src/config/networks.ts

```diff
@@ -0,0 +1,10 @@
+import { ChainIds } from '@sovryn/ethers-provider';
+
+export const networkMap = {
```

Comment:
Do we want to include a map of default gas settings for each chain as well?

Reply:
I think that's a good idea @soulBit :)

Reply:
You mean gas prices or gas limit for each transaction?
Gas prices should be always retrieved from the block before transaction is being submitted and we will be able to estimate gas limits here because new tx flow asks first tx to be confirmed before second is submitted

Reply:
I think soulBit meant gas limit, it doesn't make sense to store gas prices.

Reply:
Got it, in any case I think we don't need it here at this point.
D2 will be used for zero only at first and we will be waiting for transactions to be confirmed before sending another one (at least that's how I understand design), so it solves issue we had in D1

@tiltom @soulBit